### ISSUE TITLE
About layout issues

### ISSUE THREAD

**Issue by javanoend** [felixhayashi/TW5-TiddlyMap#360]
Hi

![image](https://user-images.githubusercontent.com/31382707/83716156-aba55880-a661-11ea-94bc-5c7cb81cda43.png)
In the case of selecting hubsize and LR and then saving and exiting, the view will become like this
![image](https://user-images.githubusercontent.com/31382707/83716176-c1b31900-a661-11ea-96d3-15ab980bfcc2.png)
Then I change the parameter of levelSeparation
![image](https://user-images.githubusercontent.com/31382707/83716282-0b9bff00-a662-11ea-9581-62bdb3eb80f1.png)
the view will become like this
![image](https://user-images.githubusercontent.com/31382707/83716315-253d4680-a662-11ea-96d2-f7b9c6afb23e.png)
Check the layout settings again 
![image](https://user-images.githubusercontent.com/31382707/83716373-4f8f0400-a662-11ea-9ea0-3dc4371eadda.png)
The parameters changed by themselves
![image](https://user-images.githubusercontent.com/31382707/83716529-bdd3c680-a662-11ea-86c9-6e0f68ec7728.png)
Changing the value of levelSeparation is not used when choosing hubsize and LR

**Comment by StudioTriple** [comment]
Hello, I have the same issue. Even more, each time I open the "edit view" panel, hierarchical is always unchecked, although my view is hierarchical.

**Comment by flibbles** [comment]
Yeah, vis-network's self-rolled configuration menu is pretty bad about stuff like this. I don't actually think I can fix it, and vis-network hasn't been maintained in years.

The best I can offer you is I'm currently in the process of writing a replacement for TiddlyMap which will use a newer engine, and be far more configurable and versatile. Just give me a couple months :P

**Comment by StudioTriple** [comment]
Oh interesting, thanks! Seems indeed unsolvable. I'm very curious about your replacement, then! It might help me solve some other of my issues :3
Thanks a lot for your work.
